torq.q

```diff
@@ -6,6 +6,11 @@ loaded:1b
 // Initialised flag - used to check if the process is still initialisation
 initialised:0b
 
+// function to add functions to initialisation list
+initlist:()
+addinitlist:{[x]if[1<count x;.proc.initlist,:enlist x]};
```

Comment:
What is the reason for the 1<count x check?  Which case is that catching?

Reply:
I think it was to make sure a function was passed with at least an argument. I don't think it's very good though, I can remove that.